File: prescriptions-backend/prescription/Entities/prescription.cs
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using MongoDB.Bson;
using System.Xml.Serialization;

namespace prescription.Entities
{
    public class Prescription
    {

        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string prescriptionId { get; set; }
        public string prescriptionDate { get; set; }
        public string patientId { get; set; }
        public string patientName { get; set; }
        public string doctorName { get; set; }
        public string patientPhoneNumber { get; set; }
        public string doctorphoneNumber { get; set; }
        public string[] symptoms { get; set; }
        public string remarks { get; set; }
        public string[] suggestedTests { get; set;}
        public List<prescribedMedicines> allPrescribedMedicines { get; set; }
    }
}

File: cure-assist-front-end/src/app/models/prescription.ts
export class Prescription {
  prescriptionId: string;
  prescriptionDate: string;
  patientId: string;
  patientName: string;
  patientPhoneNumber: string;
  doctorName: string;
  doctorphoneNumber: string;
  symptoms: string[];
  remarks: string;
  suggestedTests: string[];
  allPrescribedMedicines: PrescribedMedicines[];
}

export class PrescribedMedicines {
  medicineName: string;
  prescribedDosage: string;
  prescribedTimings: string[];
  prescribedDays: string;
}
